src/languages/dart.js

```diff
@@ -221,6 +224,42 @@ export default function(hljs) {
     $pattern: /[A-Za-z][A-Za-z0-9_]*\??/
   };
 
+  const CAPITALIZED_BUILT_IN_TYPES = BUILT_IN_TYPES.filter(type => /^[A-Z]/.test(type));
+
+  const BUILT_IN_TYPES_REGEX = `\\b(${CAPITALIZED_BUILT_IN_TYPES.join('|')})\\b`;
+
+  const CLASS_NAME_RE = regex.either(
+    /\b([A-Z]+[a-z0-9]+)+/,
+    // ends in caps
+    /\b([A-Z]+[a-z0-9]+)+[A-Z]+/,
+  );
+
+  const CLASS_REFERENCE = {
+    relevance: 0,
+    variants: [
+      {
+        // prevent built-in types with capital letter from being selected as title.class
+        match: BUILT_IN_TYPES_REGEX,
+        skip: true
```

Comment:
Why skip?

Reply:
Why wouldn't a built-in class also get `title.class`?  That or perhaps `built_in` I'd think...

Reply:
Using this snippet as an example:
```dart
class Person {
  String name;
  int age;

  Person(this.name, this.age);
}
```
Currently the library already marks String as `built_in`, so I added this regex to prevent String from being matched as `title.class`. I may have misunderstood what skip is used for (still learning how the library works) - I set `skip: true` because otherwise String wasn't decorated at all. I thought we wanted to preserve the current behavior, but if it's ok to use `title.class` for built-in classes I can remove this part

Reply:
No, without the skip String is not decorated at all. I've already removed this part because I understood it was ok to mark String with title.class, but if you want I can add it again